### PR TITLE
Use crispy forms in contest view

### LINKED ISSUE #141: Use django-crispy-forms
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [X] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in our slack channel at https://opensourcehelp.slack.com/. Request invite [here](https://opensourcehelp.herokuapp.com)

**Description**:

Replace normal form tag with django-crispy-forms. Mainly /contests/.

### PR DESCRIPTION
**Pull request by alexpnt**
**Checklist**

- [x] My branch is up-to-date with the upstream `develop` branch.
- [x] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: 

Use crispy forms in contest view.
Fixes #141 

Demo:
screenshot_2017-10-23_17-43-02

### CODE REVIEW

File: oshc/main/templates/contest_edit.html [hunk 1]
```diff
@@ -1,50 +1,13 @@
...
-            <div class="col-xs-3">
-                <textarea class="form-control" id="desc" name="desc" placeholder="Tell something about it.." rows="3"></textarea>
-            </div>
-        </div>
-
-        <div class="form-group">
-            <div class="col-xs-offset-3 col-xs-8">
+    <form name="newContest" method="POST" style="margin-top:70px;" class="form-horizontal" action="{% url 'add_contest' %}">
+        {% crispy form %}
+        <div class="form-group">#}
```
Comment (vaibhavsingh97) [inline oshc/main/templates/contest_edit.html]: Remove extra `}`

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
LGTM!! Nice work 👍 
What I think is that intead of giving error of `wrong date` we should give the example with the error to help the user.  Btw I will prefer datepicker over writing date. Can You please change it to to date picker

**Comment by alexpnt** [comment]
Date pickers added !

**Comment by vaibhavsingh97** [comment]
@AlexPnt I am not able to add date as it's disabled everything after current date
image

**Review by tapaswenipathak — APPROVED** [review]
Wohoo! 🚀 Thanks for the work and review! Had to squash, so merged with commit 7aa856a4255fceeb26a69c9c6b0c685848105cc6.

**Pull request closed without merging**